### ISSUE TITLE
Change of functionality of Delay should be included in change log 

### ISSUE THREAD

**Issue by kyledyr** [nitaybz/homebridge-delay-switch#77]
Hello, it appears that last month when 3.2.1 was released, the delay functionality changed. I previously had multiple switches with a delay of 0 because I wanted it to immediately trigger a motion notification. In 3.2.1 a change was made that a delay of 0 now disables the motion detector entirely. I only noticed this weeks after the fact because my switches stopped sending notifications. I think this functionality change should be mentioned in a changelog instead of being buried in the readme file.

**Comment by mkz212** [comment]
@kyledyr 

You are right that the changes should be better described in the changelog (release notes) and I hope the author will take this request into account.

In your case, the simplest solution is to set the delay to 1 milliseconds.

**Comment by nitaybz** [comment]
Fair point about the changelog. Noted, and the behavior (delay 0 disables the sensor) is documented in the README.

Sorry for the long delay on this one. I'm not actively using Homebridge anymore, so this is more of an acknowledgment than a fix promise.
